plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -878,14 +878,23 @@ p=. Reaction effect syntax
 
 Example: When a transition or local reaction specified as @ev1 [x > 3] / x += 1; notify(x); raise ev2@ is executed, it increments variable _x_ by 1, calls the _notify_ operation with _x_ as parameter, and raises event _ev2_.
 
+
 h4(#raising-and-processing-an-event). Raising and processing an event
 
-For the syntax of event raising, see "Raising an event":#raising-an-event.
-For the syntax of event declaration, see "Events":#events.
+For the syntax of event raising, see section "&quot;Raising an event&quot;":#raising-an-event.
+For the syntax of event declaration, see section "&quot;Events&quot;":#events.
+
+When a reaction raises an event, the subsequent behaviour of the state machine is highly dependent on the execution scheme, selected by either the "@CycleBased":#statechart_language_cyclebased (default) or the "@EventDriven":#statechart_language_eventdriven annotation. In short:
+* In the _cycle-based execution scheme_, the raised event will be added to the events that are processed by the current run-to-completion step. However, it will only be visible "downstream" in the run cycle, i.e. active states that have not yet been executed can take the event into account. When the run cycle is completed, the event will cease to exist. It can thus not be processed in any subsequent run-to-completion step.
```

Comment:
This is a quite usual term. If you don't agree, you are welcome to attribute it to poetic license.